examples/Utils/Feeds.php

```diff
@@ -0,0 +1,178 @@
+                'No matching feed attribute for feed item attribute ID: '
+                . $targetFeedItemAttributeValue->getFeedAttributeIdUnwrapped()
+            );
+        }
+
+        return $attributeIndex;
+    }
+
+    /**
+     * Retrieves the place holder fields to field attributes map for a flights feed.
+     *
+     * @see Feeds::placeholderFieldsMapFor()
+     *
+     * @param string $feedResourceName the feed resource name to get the attributes from
+     * @param int $customerId the customer ID
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @return array the map from placeholder fields to feed attributes
+     */
+    public static function flightPlaceholderFieldsMapFor(
+        string $feedResourceName,
+        int $customerId,
+        GoogleAdsClient $googleAdsClient
+    ) {
+        return self::placeholderFieldsMapFor(
+            $feedResourceName,
+            $customerId,
+            $googleAdsClient,
+            function (FeedAttribute $feedAttribute) {
+                switch ($feedAttribute->getNameUnwrapped()) {
+                    case 'Flight Description':
+                        return FlightPlaceholderField::FLIGHT_DESCRIPTION;
+                        break;
+                    case 'Destination ID':
+                        return FlightPlaceholderField::DESTINATION_ID;
+                        break;
+                    case 'Flight Price':
+                        return FlightPlaceholderField::FLIGHT_PRICE;
+                        break;
+                    case 'Flight Sale Price':
+                        return FlightPlaceholderField::FLIGHT_SALE_PRICE;
+                        break;
+                    case 'Final URLs':
+                        return FlightPlaceholderField::FINAL_URLS;
+                        break;
+                    // See FlightPlaceholderField.php for all available values.
+                    default:
+                        throw new \RuntimeException('Invalid feed attribute name.');
+                }
+            }
+        );
+    }
+
+    /**
+     * Retrieves the place holder fields to field attributes map for a feed. The initial
+     * query retrieves the feed attributes, or columns, of the feed. Each feed attribute will also
+     * include the feed attribute ID, which will be used in a subsequent step.
+     *
+     * The example then inserts a new key-value pair into a map for each
+     * feed attribute, which is the return value of the method:
+     * - The keys are the placeholder types that the columns will be.
+     * - The values are the feed attributes.
+     *
+     * @param string $feedResourceName the feed resource name to get the attributes from
+     * @param int $customerId the customer ID
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param callable $mappingFunction the function that maps feed attribute names to placeholder
```

Comment:
Change for a map, related to another comment above

Reply:
See my comment above. I don't think using a map is more convenient here.
At the end of this method, the callable is needed.